docs/workflows/templates.md

```diff
@@ -0,0 +1,60 @@
+## Content templates
+
+## Background
+
+If a user wants to customize their systems with images consisting of different repository versions, content templates
```

Comment:
I suspect we should capitalize Content Template across the doc, but i'll let @swadeley  weigh in

Reply:
Yes, you should, but main thing is to be consistent.